components/brave_wallet/common/config.gni

```diff
@@ -12,5 +12,5 @@ declare_args() {
   enable_zcash_by_default = !is_ios && !is_android
 
   # Flag is used to allow future size difference calculation
-  enable_orchard = false
+  enable_orchard = true
```

Comment:
why are we changing this buildflag as part of this PR? I don't think it makes sense to mix concerns like this and also now that we're past the big initial commit I don't want to separate adding rust crates from the code that uses them. I want to see how this is going to be used before I approve adding the crates.

Reply:
Usage of this crate is done in this PR: https://github.com/brave/brave-core/pull/23076